src/QuartoNotebookWorker/src/ojs_define.jl

```diff
@@ -8,4 +8,12 @@ function _ojs_define(::Any, kwargs)
 end
 
 ojs_convert(kwargs) = [Dict("name" => k, "value" => _ojs_convert(v)) for (k, v) in kwargs]
-_ojs_convert(x) = x
+
+function _ojs_convert(x)
+    if Tables.isrowtable(x)
```

Comment:
Is `isrowtable` right? Because currently that breaks usage of `DataFrame` objects here since

```
julia> df = DataFrame(a = 1:2)
2×1 DataFrame
 Row │ a
     │ Int64
─────┼───────
   1 │     1
   2 │     2

julia> DataFrames.Tables.isrowtable(df)
false

julia> DataFrames.Tables.istable(df)
true

julia> collect(DataFrames.Tables.rows(df))
2-element Vector{DataFrameRow}:
 DataFrameRow
 Row │ a
     │ Int64
─────┼───────
   1 │     1
 DataFrameRow
 Row │ a
     │ Int64
─────┼───────
   2 │     2
```

I'd assume that `istable` is sufficient.

Reply:
Yes, that's right, thanks for noticing, I'll fix it.